app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/settings/SettingsActivity.kt

```diff
@@ -21,7 +25,7 @@ class SettingsActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_settings)
+        setContentView(org.wikiedufoundation.wikiedudashboard.R.layout.activity_settings)
```

Comment:
org.wikiedufoundation.wikiedudashboard.R can be imported in top to make it more clean.